src/utils/updates.ts

```diff
@@ -20,7 +20,12 @@ const watchGovernance = async (
   const f = makeFollower(dappConfig.GOVERNANCE_KEY, leader, { unserializer });
 
   for await (const { value } of iterateLatest(f)) {
-    setGovernedParams(value);
+    const current = value.current;
+    const GiveStableFee = current.GiveStableFee.value;
```

Comment:
blocker: do not strip the brands. one it's confusing to have `GiveStableFee` mean an amount or a bigint. But more importantly it's a discipline that prevents a class of nasty bugs

Reply:
`value` is actually a `Ratio` here (kind of confusing), so this is what we want. It's not pulling out the `value` from an `Amount`. Similar for `MintLimit`, `value` is an `Amount` not a `bigint`.

Reply:
oh right, this is `.value` of the Governed Param type, not Amount. 👍